src/Model/Common/StatusInformation.php

```diff
@@ -20,6 +20,7 @@ class StatusInformation
      * of returned status codes is contained in the list.
      *
      * @var int $statusCode
```

Comment:
annotate as `int|string`, both seems to be true depending on environment

Reply:
The purpose of the pull request was the fix in getStatusCode() for non-int values in $statusCode causing downstream exceptions.

We would suggest for you to update the phpdoc documenting the design intent of what type this property should have
as it is not our place to speculate about design intentions.

Reply:
This is not about application design. According to your observation, the property is populated inconsistently by the PHP SOAP module or any other language feature. To reflect this and to satisfy code quality checks, the type annotation must be updated to document the actual possible types. The library does not and cannot control the data type. It is auto-populated from the web service response.

Reply:
okay. will update.